src/actions/contact-request/saveContactRequest.ts

```diff
@@ -45,6 +46,14 @@ export const saveContactRequest = async ({
         await deleteSavedContactRequest(senderEmail, profileId)
         throw Error('Failed to send contact request')
       }
+      try {
+        await mailerliteClient.addSubscriberToMailerLite(
```

Comment:
**nit:** no need for creating another try catch block. Let's put it into previous `try`